### DISCUSSION TITLE
求助laydate   mark

### DISCUSSION THREAD

**Discussion by bloodstain** [layui/layui#1733] [问答]
let  mark_obj1 = {
    "2024-3-1":"1<div>缺失</div>",
    "2024-3-3":"1<div>缺失</div>",
    "2024-3-4":"1<div>缺失</div>",
    "2024-3-7":"1<div>缺失</div>"
}
let  mark_obj2 = {
    "2024-2-1":"1<div>异常</div>",
    "2024-2-3":"1<div>异常</div>",
    "2024-2-4":"1<div>异常</div>",
    "2024-2-7":"1<div>异常</div>"
}
let option = {
    elem: '#static-calender',
    position: 'static',
    showBottom: false,
    format:'yyyy-MM-dd',
    id:"test",
    mark: mark_obj1,
    ready:function(){
        console.log(111);
    },
    // holidays: [[],holidays_data],
    done:function(value,date){
    },
    change:function(value,date){
    }
}  
有这个日期控件  我想点击上一月 让mark 得自定义数据展示在日期面板上并展示 求大佬帮忙看看

**Comment by Sight-wcg** [comment]
目前不支持，你可以基于这个 discussions 创建一个新的 issues 吗？方便跟踪问题，将来的版本会支持。

**Reply by bloodstain** [reply]
好的，我去提一个issues
